var/spack/repos/builtin/packages/libxsmm/package.py

```diff
@@ -13,10 +13,11 @@ class Libxsmm(MakefilePackage):
     and small convolutions.'''
 
     homepage = 'https://github.com/hfp/libxsmm'
-    url      = 'https://github.com/hfp/libxsmm/archive/1.9.tar.gz'
+    url      = 'https://github.com/hfp/libxsmm/archive/1.10.tar.gz'
     git      = 'https://github.com/hfp/libxsmm.git'
 
     version('develop', branch='master')
+    version('1.10',  '1ad21021739211715649f449957fb872')
```

Comment:
could you provide the sha256 hash?

Reply:
I will update the PR and replace all MD5 sums with SHA256.